crates/sui-network/src/randomness/mod.rs

```diff
@@ -232,6 +233,7 @@ impl RandomnessEventLoop {
                 Ok(acc)
             },
         )?);
+        self.blacklisted_peers = BTreeSet::new();
```

Comment:
optional nit just to keep things organized - I'm trying to keep this roughly in order that the struct members appear in the struct definition, while you're in this area can you maybe move this line and the one above it below the update to `self.authority_info`?